Formula/sdl.rb

```diff
@@ -47,8 +47,7 @@ class Sdl < Formula
   end
 
   livecheck do
-    url "https://www.libsdl.org/release/"
-    regex(/href=.*?SDL[._-]v?(\d+(?:\.\d+)+)\.t/i)
+    skip "SDL 1.2 is deprecated upstream"
```

Comment:
Then we should use `deprecate!`

Reply:
Works for me. To make this easy to merge, I'm going to pare this PR down to the `sdl2` `livecheck` block commits and handle the `sdl` deprecations in a separate PR.